src/Cache/Engine/FileEngine.php

```diff
@@ -413,8 +411,16 @@ protected function _setKey($key, $createKey = false)
      *
      * @return bool
      */
-    protected function _active()
+    protected function _dirWritable()
```

Comment:
protected methods can be extended outside the framework and userland code will break if methods are renamed in minor or bugfix releases. BC can only be broken in major versions.